modules/apps/27-interchain-accounts/types/metadata.go

```diff
@@ -33,11 +33,11 @@ func NewMetadata(version, controllerConnectionID, hostConnectionID, accAddress,
 }
 
 // NewDefaultMetadata creates and returns a new ICS27 Metadata instance containing the default ICS27 Metadata values
-// with the provided controller and host connection identifiers
-func NewDefaultMetadata(controllerConnectionID, hostConnectionID string) Metadata {
```

Comment:
We could also just leave this function and `NewDefaultMetadataString` as they are, with two parameters, and if any implementation cannot provide a host connection ID, then they can just pass an empty string.